be/src/storage/lake/pk_index_loader.h

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <condition_variable>
+#include <cstdint>
+#include <map>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <vector>
+
+#include "column/vectorized_fwd.h"
+#include "common/statusor.h"
+#include "storage/lake/types_fwd.h"
+
+namespace starrocks::lake {
+
+class MetaFileBuilder;
+
+struct ChunkInfo {
+    ChunkUniquePtr chunk;
+    uint64_t rssid;
+    std::vector<uint32_t> rowids;
+};
+
+struct PkSegmentScanTaskContext {
```

Comment:
make it nested class of PkIndexLoader if no one else is expected to access and use this class/structure.

Reply:
done